runtime/near-vm-runner/src/logic/logic.rs

```diff
@@ -3724,6 +3845,45 @@ bls12381_p2_decompress_base + bls12381_p2_decompress_element * num_elements`
         Ok(res? as u64)
     }
 
+    /// Write the protocol storage configuration parameter value for `storage_amount_per_byte` to
+    /// the location defined by `balance_ptr`.
+    ///
+    /// # Cost
+    ///
+    /// `base` - the base cost for a simple host function call
+    /// `write_memory_base` + 16 * `write_memory_byte` - the cost of writing a `u128` to guest memory
+    pub fn storage_config_byte_cost(&mut self, balance_ptr: u64) -> Result<()> {
```

Comment:
Similarly to the `promise_return_length` it might be quite unsatisfying if we end up needing to eventually add a host function for every single different configuration variable/fee that somebody might want to calculate with in their contract.

Could this be a a single function that takes the description of the fee to write into memory via an argument?